planetaryimage/image.py

```diff
@@ -26,10 +29,25 @@ def open(cls, filename):
         filename : string
             name of file to read as an image file
         """
-        with open(filename, 'rb') as fp:
-            return cls(fp, filename)
+        if filename.endswith('.gz'):
+            try:
+                with gzip.open(filename, 'rb') as fp:
+                    return cls(fp, filename, compression='gz')
+            except AttributeError:
+                raise IOError('GZIP compression not supported until Python 2.6.')
```

Comment:
gzip with context support is available from 2.7 and bz2 from 3.3. So, gzip won't be available till 2.6 and bz2 till 2.7.